src/EnergyPlus/DXCoils.cc

```diff
@@ -10361,14 +10368,14 @@ namespace DXCoils {
         PLRHeating = 0.0;
         DXCoil(DXCoilNum).HeatingCoilRuntimeFraction = 0.0;
         // Initialize crankcase heater, operates below OAT defined in input deck for HP DX heating coil
-        if (OutdoorDryBulb < DXCoil(DXCoilNum).MaxOATCrankcaseHeater) {
+        if (CompAmbTemp < DXCoil(DXCoilNum).MaxOATCrankcaseHeater) {
```

Comment:
So here I think the change is correct.

Reply:
Turns out this was incorrect. Must compare to condenser inlet temp. Corrected in #7483